### PR TITLE
Fixed build when used in Github Actions

### PR DESCRIPTION
**Pull request by BitsAndDroids**
Out of nowhere, Bindgen was having issues parsing the issue: #19 
```cpp 
#include "Windows.h"
```
header file caused failures downstream when defining the types. 
By changing it to:

```cpp
#include <windows.h>
``` 
All seems to work again.
Updated the Bindgen version and replaced a deprecated callback function

This PR also includes the fix for the example "update on change." It could cause a segfault due to an inappropriate pointer arrhythmia. issue: #16

@Sequal32

### CODE REVIEW

**Comment by BitsAndDroids** [comment]
@Sequal32 If you want help to maintain this repo, I'm down to help you out. If interested, you can reach me at info@{myusername}.com (to avoid bots scraping the address).

File: build.rs [hunk 1]
```diff
@@ -66,6 +66,7 @@ fn main() {
         .allowlist_var("UNKNOWN_GROUP")
         .allowlist_var("SIMCONNECT_CLOUD_STATE_ARRAY_WIDTH")
         .allowlist_var("SIMCONNECT_CLOUD_STATE_ARRAY_SIZE")
+        .allowlist_type("HANDLE")
```
Comment (Sequal32) [inline build.rs:69]: What's the reason for adding this?

File: libsrc/include/SimConnect.hpp [hunk 2]
```diff
@@ -4,17 +4,17 @@
 //
 //-----------------------------------------------------------------------------
 
-#include "Windows.h"
+#include <windows.h>
```
Comment (Sequal32) [inline libsrc/include/SimConnect.hpp:7]: Please remove all formatting changes and make this lowercase change the only diff in the file.

Also for the record, [this stack overflow answer](https://stackoverflow.com/a/15466951) also provides reasoning and supports the change.

**Review by Sequal32 — CHANGES_REQUESTED** [review]
Hey @BitsAndDroids thanks so much for your contribution again, and sorry for the delay! I'm going to be active enough to maintain this repo again, and feel free to keep submitting PRs. I'll hopefully be able to get to them quicker.

**Comment by BitsAndDroids** [comment]
> Hey @BitsAndDroids thanks so much for your contribution again, and sorry for the delay! I'm going to be active enough to maintain this repo again, and feel free to keep submitting PRs. I'll hopefully be able to get to them quicker.

I'm just glad to see that you're doing well. I also implemented the changes and changed the version (0.4.0 wasn't semver).

**Review by Sequal32 — APPROVED** [review]

**Pull request merged**